web/src/i18n/locales/en/activities/goals.yaml

```diff
@@ -1,16 +1,19 @@
-title: Performance Measures and Benchmarks
+title: Performance Goals and Benchmarks
```

Comment:
Benchmarks --> Measures

Reply:
Nevermind. I spent some time thinking about this, and the definition for benchmark works better here than measure.